exercises/binary/binary_test.go

```diff
@@ -17,12 +17,12 @@ import (
 // Also define a testVersion with a value that matches
 // the targetTestVersion here.
 
-const targetTestVersion = 1
+const targetTestVersion = 2
 
 var testCases = []struct {
 	binary   string
 	expected int
-	ok bool
+	ok       bool
```

Comment:
probably good to put in a separate commit since this change is not to do with what  is wanted in #283 .